### PR TITLE
Fix #1 Add pkcli.service.db and supporting modules

### LINKED ISSUE #1: Basic db_importer
- Watch for summary files
- Add to db
- pkasyncio event loop

### PR DESCRIPTION
**Pull request by robnagler**
Support #7 added db_api for runs_by_date_and_values and run_kinds_and_value_names

### CODE REVIEW

**Comment by robnagler** [comment]
Depends on https://github.com/radiasoft/pykern/pull/510

File: slactwin/db_api.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+"""
+
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdlog, pkdp
+import datetime
+import slactwin.quest
+
+
+class DbAPI(slactwin.quest.API):
+    async def api_run_kinds_and_values(self, api_arg):
```
Comment (e-carlin) [inline slactwin/db_api.py:14]: I think api_arg needs to be documented.

**Review by e-carlin — CHANGES_REQUESTED** [review]

File: slactwin/db_query.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+    def _init_tables(self, models, method):
+        for a in inspect.getfullargspec(method).args:
+            if v := models.get(a):
+                yield a, v.table
+
+    def __call__(self, db, **kwargs):
+        return self._method(self, db, **self._tables, **kwargs)
+
+    def _query_runs_by_date_and_values(
```
[the remark this answers is not in the document]
Comment (robnagler) [reply slactwin/db_query.py:51]: https://github.com/slaclab/slactwin/issues/10

File: slactwin/db_api.py (hunk 1 above)
Comment (robnagler) [reply slactwin/db_api.py:14]: https://github.com/slaclab/slactwin/issues/10

File: slactwin/db.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        debug=(False, bool, "turn on sqlalchemy tracing"),
+        uri=pykern.pkconfig.RequiredUnlessDev(
+            None,  # "postgresql://vagrant@/slactwin",
+            _uri,
+            "sqlalchemy create_engine uri, e.g. postgresql://vagrant@/slactwin",
+        ),
+    )
+    # TODO(robnagler): need to set connection args, e.g. pooling
+    _engine = sqlalchemy.create_engine(_cfg.uri, echo=_cfg.debug)
+    _models = slactwin.db_model.init_by_db(_engine)
```
[the remark this answers is not in the document]
Comment (robnagler) [reply slactwin/db.py]: [c842d98](https://github.com/slaclab/slactwin/pull/8/commits/c842d9829858c4f13db15fdf4a5cdd38ab4d2808)

**Pull request merged**